CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+### Added
+- Added `nix::sys::socket::ControlMessage::ScmTimestamp`
```

Comment:
I could add `nix::sys::socket::sockopt::ReceiveTimestamp` was added, otherwise a binding for a particular *socket control message* should be straightforward to someone looking for the functionality of the same name.

I didn't see any other added changelog entries that added more than the shared name of the new binding, so I'm not sure what level of detail is appropriate.